### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GoodGradesBoy**
replaced nonexisting star bug command with star suggest

### CODE REVIEW

**Review by ghostlylake — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # **Starboard Discord Bot :0**
 
 ## This repository is for bugs, issues or suggestions about the bot.
-### Alternatively, you can use the star bug command.
+### Alternatively, you can use the star suggest command.
```
Comment (lewisfm) [inline README.md:4]: Maybe add code formatting to "star suggest"?
```suggestion
### Alternatively, you can use the `star suggest` command.
```

Comment (GoodGradesBoy) [reply README.md:4]: no, that's bad

Comment (doinkythederp) [reply README.md:4]: ok whatever

**Review by lewisfm — APPROVED** [review]
looks good to me

**Review by lewisfm — APPROVED** [review]
still looks good to me

**Pull request merged**